lib/driverProviders/local.ts

```diff
@@ -182,10 +171,12 @@ export class Local extends DriverProvider {
    * @return {q.promise} A promise which will resolve when the environment
    *     is down.
    */
-  teardownEnv(): q.Promise<any> {
-    return super.teardownEnv().then(() => {
+  async teardownEnv(): Promise<any> {
+    try {
+      await super.teardownEnv();
       logger.info('Shutting down selenium standalone server.');
-      return this.server_.stop();
-    });
+      await this.server_.stop();
+    } catch (_) {
```

Comment:
Why eat the exception here? We could just leave out the try {} catch {} and let the uncaught exception handler take care of it if there's nothing to do here. Or at least log the error.

Reply:
Neat, I deleted this method.